### PR TITLE
Create GUIDE_ru

### PR DESCRIPTION
**Pull request by AbdulKhadhar**
Add Language Russian

### CODE REVIEW

**Comment by AbdulKhadhar** [comment]
Adding all 6 UN Languages makes the guides more social 😊
Russian and French are also among them

File: GUIDE_ru [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+
+Цифровые данные означают данные, которые в конечном итоге хранятся в двоичном формате, то есть в виде нулей и единиц, поскольку сами компьютеры управляются двоично - электрическими сигналами, которые либо «включены», либо «выключены», что соответствует 1 или 0, и поэтому двоичные данные гораздо проще для компьютеров, чтобы понять, чем любой другой.
+
+Читаемые человеком метаданные, хранящиеся в начале каждого ролика, включают в себя информацию о самом фильме, руководство по используемому QR-кодированию, программу для его декодирования и указатель. Индекс перечисляет заголовок, номер начального кадра и контрольную сумму для каждого файла, сохраненного на этом барабане.
+
+Файл представляет собой единый связный объект данных. Контрольная сумма - это уникальное значение из расчета, известное как хеш-функция, которое выполняется по всему содержимому файла, чтобы гарантировать, что его содержимое не было повреждено или повреждено; хеш-функция, используемая в архиве, известна как SHA-1.
```
Comment (Maxim-Mazurok) [inline GUIDE_ru:47]: "повреждено или повреждено" means "damaged or damaged", which doesn't sound right.
It should be "повреждено или искажено".

File: GUIDE_ru [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+
+Каждый QR-код состоит из поля крошечных белых или черных квадратов, которые занимают почти весь кадр фильма. Мы используем QR-коды, потому что они намного более компактны и надежны, чем текст, читаемый человеком. QR-код декодирует в двоичные данные, то есть серию единиц и нулей.
```
Comment (Maxim-Mazurok) [inline GUIDE_ru:49]: "почти весь кадр фильма" means "almost the entire frame of movie". 
It should be "почти весь кадр плёнки"

File: GUIDE_ru [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+
+После списка файлов пользовательских данных приведен список местоположений цифровых данных. Этот список включает в себя идентификатор файла, начальный кадр, начальный байт, конечный кадр и конечный байт. Таким образом, используя гипотетический пример CPython, элемент в этом списке с идентификатором 12345 может иметь начальный кадр 054321, начальный байт 03210321, конечный кадр 054545 и конечный байт 12321232.
+
+Это означает, что для получения данных CPython: перейдите к кадру 54321 этого фильма. Декодировать все кадры от начального кадра 54321 до конечного кадра 54545 в двоичные значения с помощью средств, описанных ниже. Это даст вам 225 фрагментов данных, пронумерованных от 54321 до 54545, которые начнутся с набора пустых фрагментов без данных. Откажитесь от первых 3210320 байтов в первом непустом фрагменте данных. Добавьте все «средние» фрагменты данных по порядку. Наконец, добавьте первые 12321232 байта из последнего фрагмента данных, 54545. Теперь вы собрали полный репозиторий CPython в виде одного сжатого архивного файла.
```
Comment (Maxim-Mazurok) [inline GUIDE_ru:107]: Probably all mentions of "фильм" (movie) should be replaced with "плёнка" (a thin flexible strip of plastic or other material coated with light-sensitive emulsion for exposure in a camera, used to produce photographs or motion pictures - QR codes in our case)

File: GUIDE_ru (hunk 2 above)
Comment (AbdulKhadhar) [reply GUIDE_ru]: 👍

File: GUIDE_ru (hunk 3 above)
Comment (AbdulKhadhar) [reply GUIDE_ru]: 👍

**Comment by Maxim-Mazurok** [comment]
TBH, it would be easier to translate for someone who knows Russian. I see no point translating it using google translate. But good intention though 👍

**Comment by AbdulKhadhar** [comment]
> 
> 
> TBH, it would be easier to translate for someone who knows Russian. I see no point translating it using google translate. But good intention though 👍

🙂👍 Thanks for the Review

**Comment by rezendi** [comment]
Thanks for the PR. We very much appreciate your work, but at least at the moment, we're limiting the translations of the Guide to professional translations. We may well revisit this in future and move to a more expansive list of crowdsourced translations, however, in which case this PR will be reopened and reexamined.

**Pull request closed without merging**